packages/rancher-sriov/generated-changes/overlay/templates/secrets.yaml

```diff
@@ -0,0 +1,20 @@
+{{- if not .Values.cert_manager -}}
+{{- if .Values.operator.enableAdmissionController }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: operator-webhook-service
```

Comment:
Should the naming scheme for these be based on `{{ include "sriov-network-operator.fullname" . }}`?

Reply:
I did not want to add another delta with regards to upstream. I'd suggest we wait until upstream implements this

Reply:
Same as above. If you believe this is important, I could spend time adding new functions in _helpers but I don't see the benefit